pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -204,14 +205,43 @@ type inode struct {
 	kernfs.InodeDirectoryNoNewChildren
 	kernfs.OrderedChildren
 
-	locks vfs.FileLocks
-
+	NodeID uint64
 	dentry kernfs.Dentry
+	locks  vfs.FileLocks
+
+	// the owning filesystem. fs is immutable.
+	fs *filesystem
+
+	// size of the file.
+	size uint64
```

Comment:
Seem that this variable is always zero?

Reply:
So far, yes. But write_update_size https://elixir.bootlin.com/linux/v4.4/source/fs/fuse/file.c#L972 and read_update_size https://elixir.bootlin.com/linux/v4.4/source/fs/fuse/file.c#L651 would change it (when we implement read and write)

Reply:
The OPEN code used `size` in its check. No one can read or write before open. So that check will always check a zero value? I suppose an update to `size` should be added to the request before OPEN, i.e. GET_ATTR?